src/mm-file-upload/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<script language="javascript" src="../../bower_components/webcomponentsjs/webcomponents-lite.js"></script>
+	<link rel="import" href="../../bower_components/polymer/polymer.html"/>
+	<link rel="import" href="mm-file-upload.html">
+</head>
+<body>
+	<h3>Regular mm-file-upload</h3>
+	<mm-file-upload></mm-file-upload>
+	<hr />
+	<h3>Disabled</h3>
+	<mm-file-upload disabled></mm-file-upload>
+	<hr />
+	<h3>Drag-and-drop enabled</h3>
+	<mm-file-upload drag-drop-enabled accept="*.go"></mm-file-upload>
```

Comment:
When I drag-n-drop "hello.go" to the input, the field is marked as invalid. With no `accept` specified, it's valid. Looks like an issue with the validation regex?